tests/unit.test.ts

```
@@ -0,0 +1,478 @@
+describe('#67 sanitizeForTemplate', () => {
+  it('escapes backticks and backslashes', () => {
+    expect(sanitizeForTemplate('a`b\\c')).toBe('a\\`b\\\\c');
+  });
+
+  it('escapes ${ template-expression sequences', () => {
+    const malicious = 'desc ${process.env.SECRET}';
+    const out = sanitizeForTemplate(malicious);
+    expect(out).toBe('desc \\${process.env.SECRET}');
+    // When embedded in a real template literal, the expression is inert.
+    // eslint-disable-next-line no-eval
+    const rendered = eval('`' + out + '`');
```

Biome (2.4.16)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)